app/src/main/cpp/src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_received_policy_update.cc

```diff
@@ -63,11 +63,19 @@ void OnReceivedPolicyUpdate::Run() {
   const std::string& file_path =
       (*message_)[strings::msg_params][hmi_notification::policyfile].asString();
   policy::BinaryMessage file_content;
-  if (!file_system::ReadBinaryFile(file_path, file_content)) {
+
+  std::string file_name = file_path;
+
+#ifdef __ANDROID__
+  const std::string t_name = file_path.substr(file_path.find_last_of("/\\") + 1);
+  file_name = "/data/user/0/org.luxoft.sdl_core/cache/ivsu_cache/" + t_name;
```

Comment:
@VadymLuchko please use `application_manager_.get_settings().app_info_storage()` for getting this path instead of hardcoding it

Reply:
@AKalinich-Luxoft  fixed in 4637feb7c4f80e2386082e9a38d74f7af4dc42f9